desktop/components/follow_button/view.coffee

```diff
@@ -56,7 +56,7 @@ module.exports = class FollowButton extends Backbone.View
       mediator.trigger 'open:auth',
         mode: 'register'
         copy: "Sign up to follow #{@label}"
-        redirectTo: @href || "#{@model.href()}/follow"
+        destination: @href || "#{@model.href()}/follow"
```

Comment:
Here, I opted to update `redirectTo` to `destination`. This means we _won't_ break the server-side follows which are working. You'll be redirected here after onboarding.